ewcmd.py

```diff
@@ -707,3 +707,18 @@ def weather_txt(id_server):
 		response = "You lack the moral fiber necessary for urination."
 
 	await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
+
+"""find out how many days are left until the 31st"""
+async def fursuit(cmd):
+	user_data = EwUser(member=cmd.message.author)
+	mutations = user_data.get_mutations()
+	market_data = EwMarket(id_server=cmd.message.server.id)
+
+	if ewcfg.mutation_id_organicfursuit in mutations:
+		days_until = market_data.day % 31
```

Comment:
this is how many days have passed since the last full moon, not how many until the next one